cmds/import_image/import_image_cmd.go

```diff
@@ -84,5 +114,28 @@ func (cmd *ImportImageCmd) Run() error {
 	cmd.Id = vgbdtgObject.Id
 	cmd.Uuid = vgbdtgObject.GlobalIdentifier
 
+	if cmd.Public {
+		execStmtRetryable := boshretry.NewRetryable(
+			func() (bool, error) {
+				id, err := vgbdtgService.CreatePublicArchiveTransaction(cmd.Id, cmd.PublicName, cmd.PublicNote, cmd.PublicNote, locations)
+				if err != nil {
+					return true, errors.New(fmt.Sprintf("There would be an active transaction in progress."))
+				}
+
+				cmd.Id = id
```

Comment:
I raised [softlayer-go issue 103](https://github.com/maximilien/softlayer-go/issues/103) there. But according to my last comment, create_light_stemcell task retrieves all properties for the public image, including uuid. So actually it's not needed to get uuid in import_stemcell task.